### PR TITLE
Workflow updates

### PR DESCRIPTION
**Pull request by erikpena**
- Updated internal readme with more info
- Added playwright to npm-publish workflow
- Updated ci workflow to use npm ci

### CODE REVIEW

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ jobs:
         with:
           node-version: 14
       - run: npm ci
+      - uses: microsoft/playwright-github-action@v1
```
Comment (dylanjha) [inline .github/workflows/npm-publish.yml:16]: should this go up to line 13 with the other `uses` options?

Comment (erikpena) [reply .github/workflows/npm-publish.yml:16]: Yea absolutely it should.  Good catch

File: .github/workflows/npm-publish.yml [hunk 2]
```diff
@@ -9,6 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: microsoft/playwright-github-action@v1
       - uses: actions/setup-node@v1
```
Comment (dylanjha) [inline .github/workflows/npm-publish.yml:13]: I think this is good, but I noticed the order of `uses` in the CI script is

```
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v1
      - uses: microsoft/playwright-github-action@v1
```

Should we make them the same order for consistency?

Comment (erikpena) [reply .github/workflows/npm-publish.yml:13]: So I think that was how it was (effectively) before I made the last change.  On the topic of consistency, maybe what I'll do is model it closer to how we do the CI workflow though.

Comment (erikpena) [reply .github/workflows/npm-publish.yml:13]: Cool done 😎

**Review by dylanjha — APPROVED** [review]

**Pull request merged**